src/groups/mqb/mqbs/mqbs_filebackedstorage.h

```diff
@@ -256,6 +252,9 @@ class FileBackedStorage BSLS_KEYWORD_FINAL : public ReplicatedStorage {
     /// otherwise return false.
     virtual bool isPersistent() const BSLS_KEYWORD_OVERRIDE;
 
+    /// Return the queue this storage is associated with.
+    virtual mqbi::Queue* queue() const BSLS_KEYWORD_OVERRIDE;
```

Comment:
Can the queue still be null? If so, please document.

Reply:
Storage exists without a queue before setQueue call.